### PR TITLE
RFC 3: change how we include gem assets in applications

### PR DESCRIPTION
**Pull request by andysellick**
👉 [Rendered version](https://github.com/alphagov/govuk_publishing_components/blob/a29d7f94aef5046790d46909d963dc924084f5cf/docs/rfc-003.md)

### CODE REVIEW

**Comment by vanitabarrett** [comment]
Should we be putting this in the govuk-rfc repo?

**Comment by andysellick** [comment]
The past two RFCs (admittedly also written by me, so I could be generally wrong on this) were in this repo: https://github.com/alphagov/govuk_publishing_components/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed+rfc

**Comment by sihugh** [comment]
I agree with @vanitabarrett!

File: docs/rfc-003.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+// currently
+//= require govuk_publishing_components/all_components
+```
+
+```
+// proposed
+//= require govuk_publishing_components/components/accordion
+//= require govuk_publishing_components/components/button
+// (etc.)
+```
```
Comment (injms) [inline docs/rfc-003.md]: I think this would also require explicit initialisation of each component as well.

**Comment by andysellick** [comment]
@vanitabarrett @sihugh fine... https://github.com/alphagov/govuk-rfcs/pull/110

**Pull request closed without merging**